addons/ui/ACE_UI.hpp

```diff
@@ -0,0 +1,177 @@
+// IDD, elements[]s, Element (must be string to compare to changed setting name), Selective Type (0-both, 1-ground, 2-vehicle)
+class ACE_UI {
+    class weaponName {
+        idd = 300;
+        elements[] = {118};
+        condition = QUOTE(ACE_player == vehicle ACE_player);
+        class conditions {};
```

Comment:
Condition is the main condition (vehicle/soldier UI), conditions class is for anything else to add extra conditions, such as other components or mods.